src/main/java/com/alvarium/annotators/Annotator.java

```diff
@@ -0,0 +1,18 @@
+package com.alvarium.annotators;
+
+import com.alvarium.contracts.Annotation;
+import com.alvarium.utils.PropertyBag;
+
+/**
+ * A unit responsible for annotating raw data and producing an Annotation object
+ */
+public interface Annotator {
+  /**
+   * creates an Annotation from the given raw data
+   * @param ctx
+   * @param data
+   * @return Annotation object
+   * @throws AnnotatorException
+   */
+  public Annotation make(PropertyBag ctx, byte[] data) throws AnnotatorException;  
```

Comment:
the `make` method in the java sdk represents the `Do` function in go. I couldn't go with the same go naming convention as `do` is a keyword in java. I assume `Do` was chosen in the go sdk because `make` is a keyword in go.

Reply:
I think `execute` would be a better substitute. `make` is not a reserved word in Go but it does refer to the build tools as in the `Makefile`. If we needed to standardize on the API across implementations, I think `execute` is a better choice as we're asking the Annotator implementation to execute/do its verification.

So please rename accordingly to `execute`